cnxdb/tests/testing.py

```diff
@@ -13,16 +13,28 @@
     'host': 'localhost',
     'port': '5432',
     }
-_DEFAULT_CONNECTION_STRING = ' '.join(
-    ['{}={}'.format(key, value)
-     for key, value in _DEFAULT_CONNECTION_SETTINGS.items()])
-_CONNECTION_STRING = os.environ.get('TESTING_CONNECTION_STRING',
-                                    _DEFAULT_CONNECTION_STRING)
+
+
+def parse_connection_string_to_parts(connection_string):
+    """Parses a connection string to parts (dict of key values)."""
+    return dict(map(lambda x: x.split('='), connection_string.split()))
```

Comment:
In the future perhaps consider using list comprehension instead of `map`?  I think list comprehension is easier to read and faster.